src/main.py

```diff
@@ -53,6 +53,9 @@ def __init__(self):
         # initialize SLMDemo
         self.SLM = SLMDemo()
```

Comment:
Is it a demo or the worker of the actual SLM? Please clarify by the class's name